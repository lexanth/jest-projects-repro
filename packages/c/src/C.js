import React from 'react'

export const C = () => <div data-testid="C" />
