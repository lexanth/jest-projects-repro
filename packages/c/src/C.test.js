import { render } from '@testing-library/react'
import { C } from './C'
import React from 'react'

describe('C', () => {
  it('should render', () => {
    const { container } = render(<C />)
    expect(container).toMatchSnapshot()
  })
})
