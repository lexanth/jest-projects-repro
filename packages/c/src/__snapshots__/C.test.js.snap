// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`C should render 1`] = `
<div>
  <div
    data-testid="C"
  />
</div>
`;
